common/src/main/java/fi/hel/verkkokauppa/common/queue/service/SendNotificationService.java

```diff
@@ -95,4 +94,37 @@ public void sendRefundMessageNotification(
 
     }
 
+    public void sendErrorNotification(
+            String endpoint,
+            String message,
+            Exception exception
+    ) {
+        sendErrorNotification(
+                endpoint,
+                message + " " + exception.getMessage(),
+                exception.getStackTrace().toString()
+        );
+    }
+
+    public void sendErrorNotification(
+            String endpoint,
```

Comment:
This can be function also etc? message should be one string that is passed to this function. Makes this more usable in different error notification places.